src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx

```diff
@@ -230,7 +230,7 @@ The following table describes New Relic's Services in Scope of New Relic's assur
 
```

Comment:
yeah I don't know. It falls into that strange area where legal/license people add capitalized words because they reflect something in the contract. I don't really know. I didn't add this one.